static-site/src/components/ListResources/index.tsx

```diff
@@ -37,8 +38,15 @@ function ListResources({
   defaultGroupLinks=false,
   groupDisplayNames,
   showcase,
+  parserCallBackFxn,
 }: ListResourcesProps) {
-  const {groups, dataFetchError} = useDataFetch(sourceId, versioned, defaultGroupLinks, groupDisplayNames);
+  const {groups, dataFetchError} = useDataFetch(
+    sourceId,
+    versioned,
+    defaultGroupLinks,
+    groupDisplayNames,
+    parserCallBackFxn,
```

Comment:
### Nitpick: naming the callback function

Currently the caller names it `parserCallBackFxn` while the function signature uses `parserCallBack`.

Suggestion 1: use the same name. Between the two, I'd pick `parserCallBack` – the `Fxn` abbreviation is new to this codebase and I think redundant with the type info.

Suggestion 2: Based on interface name suggested in based on https://github.com/nextstrain/nextstrain.org/pull/904#discussion_r1631592549, use the name `getResourceListingInfo`. Usage would look like this, which to me reads better than the current names.

```js
// static-site/src/sections/pathogens.jsx
const getResourceListingInfo = async (response) => (await response.json()).dataset[sourceId];

// static-site/src/components/ListResources/useDataFetch.ts
({ pathVersions, pathPrefix } = await getResourceListingInfo(response));

// static-site/src/pages/groups.jsx
async function getResourceListingInfo(response) {
    const datasets = (await response.json()).datasets;
    // ...
    return { pathPrefix: "groups/", pathVersions };
}
```

Reply:
I've got a lean for callbacks to be clearly identifiable by name; additionally, this callback isn't getting, it's only parsing.

How does `parseResourceListingInfoCb` grab you?

Reply:
Better – not a huge fan of abbreviations (seems to go against "clearly identifiable by name"), but dealer's choice!

Reply:
If spelling it out, I would use `Callback` instead of `CallBack` (with `React.useCallback` in mind)

Reply:
Split the difference with `parseResourceListingCallback`.